src/applications/_mock-form-ae-design-patterns/patterns/pattern2/personal-information/config/personalInfo.js

```diff
@@ -28,9 +28,15 @@ const personalInfoMax = personalInformationPage({
     ssn: { show: true, required: true },
     vaFileNumber: { show: true, required: true },
     dateOfBirth: { show: true, required: true },
-    gender: { show: true, required: true },
+    sex: { show: true, required: true },
```

Comment:
i appreciate us keeping this in, for future us, but i want to make sure that we're not _showing_ it anywhere on the frontend, right?